src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -185,6 +200,9 @@ def _confirm_af_intentions(self) -> bool:
     def _enable_widgets(self, enable: bool) -> None:
         for child in self.children():
             if child is not self.control_btns and hasattr(child, "setEnabled"):
+                if isinstance(child, CoreMDATabs):
+                    child._enable_tabs(enable)
+                    continue
```

Comment:
rather than putting this `if` clause inside of the outer if clause, and then calling `continue`, just use if/else:

```python
            if isinstance(child, CoreMDATabs):
                child._enable_tabs(enable)
            elif child is not self.control_btns and hasattr(child, "setEnabled"):
                child.setEnabled(enable)
```